raiden/transfer/mediated_transfer/initiator.py

```diff
@@ -102,15 +102,19 @@ def try_new_route(
             reason = 'none of the available routes could be used'
 
         transfer_failed = EventTransferSentFailed(
-            identifier=transfer_description.payment_identifier,
-            reason=reason,
+            None,
+            None,
```

Comment:
None is not the correct value here

Reply:
Is there anyway I could get payment_network_identifier and channel_identifier here?

Reply:
The variable `channel_state`

Reply:
it is None: 
https://github.com/raiden-network/raiden/blob/c2b839379fffddb8d61e803581afab4cdc46bd69/raiden/transfer/mediated_transfer/initiator.py#L98

Reply:
True, I misread.

So, channel_identifer cannot be specified, the transfer failed exactly because there are no channels to complete it available.

The token network id can be found at the [transfer description](https://github.com/raiden-network/raiden/blob/784c50d5d88bb02fd6fe1558e137aa78cf33087d/raiden/transfer/mediated_transfer/state.py#L370).

The payment_network_id seems to be missing, you can add it to the `TransferDescriptionWithSecretState` state